src/DynamoRevit/DynamoRevit.cs

```diff
@@ -890,4 +912,85 @@ public override IEnumerable<string> GetDynamoUserDataLocations()
             return paths;
         }
     }
+
+    [Serializable()]
+    public class ExtensionInfo
```

Comment:
this name seems to conflict with the already existing `Extension` and `ViewExtension` mechanisms that dynamo provides for loading application level plugins.

* Also, does this need to be public?
* public properties should have /// summary comments that describe their use etc.

Reply:
Extension seems too generic, I think what we are doing here is to pre load customized packages.

Reply:
It needs to be public for XmlSerializer serializer = new XmlSerializer(typeof(InternalPackage));.
I renamed it to InternalPackage and added some comments